server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import os
+from lxml import etree
+from belga.io.feed_parsers.belga_dpa_newsml_2_0 import BelgaDPANewsMLTwoFeedParser
+from . import BelgaTestCase
+
+
+class BelgaDPANewsMLTwoTestCase(BelgaTestCase):
+    filename = 'dpa_newsml_2_0_belga.xml'
+
+    def setUp(self):
+        super().setUp()
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.normpath(os.path.join(dirname, '../fixtures', self.filename))
+        provider = {'name': 'test'}
+        with open(fixture, 'rb') as f:
+            parser = BelgaDPANewsMLTwoFeedParser()
+            self.xml_root = etree.parse(f).getroot()
+            self.item = parser.parse(self.xml_root, provider)
+
+    def test_can_parse(self):
+        self.assertTrue(BelgaDPANewsMLTwoFeedParser().can_parse(self.xml_root))
+
+    def test_content(self):
+        item = self.item[0]
+        self.assertEqual(item["guid"], "urn:newsml:dpa.com:20090101:190603-99-492251:3")
+        self.assertEqual(item["uri"], "urn:newsml:dpa.com:20090101:190603-99-492251")
+        self.assertEqual(item["version"], "3")
+        self.assertEqual(item["type"], "text")
+        self.assertEqual(str(item["versioncreated"]), "2019-06-03 13:00:01+00:00")
+        self.assertEqual(item["pubstatus"], "usable")
+        self.assertEqual(item["ednote"], '\n          ')
```

Comment:
Seems like ednote is not ingested properly. I see content in the xml file, but seems like `NewsMLTwoFeedParser.parse_item_meta` can not handle it properly.

Reply:
Thanks